WebDavServer.Infrastructure.FileStorage/Services/FileStorageService.cs

```diff
@@ -447,5 +446,9 @@ private string GetContentType(string fileName)
 
             return "text/plain";
         }
+        private bool HasInvalidChars(string directoryName)

```

Comment:
Any reason why it should be static? Im trying to learn clean architecture as I go

Reply:
This is not about `clean architecture`, but about `clean code`. Since the method does not use class members, it should be defined as static

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/static#example---static-field-and-method